vendor-extra/LiberoPatternsBundle/src/Resources/views/listing-grid.html.twig

```diff
@@ -0,0 +1,50 @@
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['listing-grid']) }) -%}
```

Comment:
is this from an update to `pattern-library`? How is the version of `pattern-library` tracked? (I would assume it's not because there is no diff in `composer.lock` or similar lock file)

Reply:
It's not at the moment, result of running `bin/update-patterns pr-78`.